tools/arriba.wdl

```diff
@@ -0,0 +1,315 @@
+            -R ~{read_through_distance} \
+            -A ~{min_anchor_length} \
+            -M ~{many_spliced_events} \
+            -K ~{max_kmer_content} \
+            -V ~{max_mismatch_pvalue} \
+            -F ~{fragment_length} \
+            -U ~{max_reads} \
+            -Q ~{quantile} \
+            -e ~{exonic_fraction} \
+            -T ~{top_n} \
+            -C ~{coverage_fraction} \
+            -l ~{max_itd_length} \
+            -z ~{min_itd_allele_fraction} \
+            -Z ~{min_itd_supporting_reads} \
+            ~{if mark_duplicates then "" else "-u"} \
+            ~{if report_additional_columns_discard then "-X" else ""} \
+            ~{if fill_gaps then "-I" else ""}
+    >>>
+
+    output {
+        File fusions = "fusions.tsv"
+        File discarded_fusions = "fusions.discarded.tsv"
+    }
+
+    runtime {
+        cpu: 1
+        memory: "~{memory_gb} GB"
+        disks: "~{disk_size_gb} GB"
+        container: "quay.io/biocontainers/arriba:2.4.0--h0033a41_2"
+        maxRetries: 1
+    }
+}
+
+task arriba_tsv_to_vcf {
+    meta {
+        description: "Convert Arriba TSV format fusions to VCF format."
+        outputs: {
+            fusions_vcf: "Output file of fusions in VCF format"
+        }
+    }
+
+    parameter_meta {
+        fusions: "Input fusions in TSV format to convert to VCF"
+        reference_fasta_gz: "Gzipped reference genome in FASTA format"
+        modify_disk_size_gb: "Add to or subtract from dynamic disk space allocation. Default disk size is determined by the size of the inputs. Specified in GB."
+    }
+
+    input {
+        File fusions
+        File reference_fasta_gz
+        String vcf = basename(fusions, ".tsv") + ".vcf"
+        Int modify_disk_size_gb = 0
+    }
+
+    Int input_size_gb = ceil(size(fusions, "GiB"))
+    Int disk_size_gb = ceil(input_size_gb) + (ceil(size(reference_fasta_gz, "GiB")) * 3) + modify_disk_size_gb
+
+    String fa = basename(reference_fasta_gz, ".gz")
+
+    command <<<
+        gunzip -dc ~{reference_fasta_gz} > ~{fa}
+        convert_fusions_to_vcf.sh \
+            ~{fa} \
+            ~{fusions} \
+            ~{vcf}
+    >>>
+
+    output {
+        File fusions_vcf = vcf
+    }
+
+    runtime {
+        cpu: 1
+        memory: "4 GB"
+        disks: "~{disk_size_gb} GB"
+        container: "quay.io/biocontainers/arriba:2.4.0--h0033a41_2"
+        maxRetries: 1
+    }
+}
+
+task arriba_extract_fusion_supporting_alignments {
+    meta {
+        description: "Extract alignments that support fusions."
+        outputs: {
+            fusion_bams: "Array of BAM files corresponding with fusions in the input file"
+            fusion_bam_indexes: "Array of BAM indexes corresponding with the BAMs in the 'fusion_bams'"
+        }
+    }
+
+    parameter_meta {
+        bam: "Input BAM format file on which to call fusions"
+        bam_index: "BAM index file corresponding to the input BAM"
+        fusions: "Input fusions in TSV format to convert to VCF"
```

Comment:
I think this meta is from another task

Reply:
Let me take another look at this in the morning. Clearly, I missed a few things. I'll reping you when I've updated.